verifact/forum/schema.py

```diff
@@ -118,7 +126,53 @@ def mutate_and_get_payload(
 
         return AnswerCreate(answer=answer)
 
+class VoteUpdate(ClientIDMutation):
+    vote = Field(VoteNode)
+
+    class Input:
+        credible = Boolean() # if null, remove vote!
+        answer_id = ID()
+
+    @login_required
+    def mutate_and_get_payload(
+        self,
+        info,
+        answer_id,
+        **kwargs
+    ):
+        credible = kwargs.get('credible',None)
+        vote=None
+        viewer = info.context.user
+        try: # control flow could have been simpler -> if exists, delete, then create. but chose to do it this way for atomicity.
+            intansid = Node.get_node_from_global_id(
```

Comment:
What do you think of using the simpler syntax?

```python
answer_pk = from_global_id(input.get("answer_id"))[1]
```

The graphql id actually contains the type information, but it would throw an error regardless if the primary key doesn't exist.

Reply:
Also, it appears that the `try` block isn't protection against an error from this call, is that right? If so, maybe move it outside the try block.

Reply:
regarding the suggested syntax - curious why `input.get("answer_id")` is preferrable to just directly using the answer_id parameter variable?

Reply:
My reasoning was that it's a little cleaner because you don't _really_ need to specify `only_type` since the type is embedded in the `global_id`.  I guess it's more a preference than anything else. I think the way you did it works too.

Reply:
oh yea no I like `from_global_id` more - I was more wondering about `from_global_id(input.get("answer_id"))[1]` vs `from_global_id(answer_id)[1]`

Reply:
Ah! Yeah, I think both work there. The only reason I put everything inside the `from_global_id` call is to keep it concise since if `answer_id` is required, there's no uncertainty about retrieving it. But I think if it reads clearer to you have a separate variable, that's 💯 cool by me.